setup.py

```diff
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python3
```

Comment:
Unfortunately, I don't think python2 ca be dropped out of the equation yet. I let a maintainer confirm.
Is python2 fully incompatible with this set of changes or the update to py3 comes for a convenience?

Reply:
I don't remember. I remember that 2020 version of setuptools has dropped python 2. I don't use python 2 since 2013.

Reply:
Not everyone is that lucky ...